.prettierignore

```diff
@@ -0,0 +1,10 @@
+node_modules
```

Comment:
Is there a way to make Prettier and Stylelint use `.eslintignore`'s config (or some other consolidation)? Seems like we'd want these three files to stay in lockstep with each other, and this could reduce some possible maintenance issues (add to one and forget to add to the others) down the road.

Reply:
I know. I started looking for this in the Prettier repo, and sadly the answer was "that's not supported". (last paragraph of PR text)
This was the beginning of that rabbithole: https://github.com/prettier/prettier/issues/8506

Reply:
Ah, good deal - thanks for looking into it, @NickyMeuleman!